redbot/cogs/admin/converters.py

```diff
@@ -7,15 +7,33 @@
 
 class SelfRole(commands.Converter):
     async def convert(self, ctx: commands.Context, arg: str) -> discord.Role:
-        admin = ctx.command.cog
+        admin = ctx.command.cog.bot.get_cog("Admin")
```

Comment:
This line is nice necessary since the SelfRole converter is a converter specifically for the admin cog

Reply:
Oh yeah whoops, I repurposed that line from old code that I was using it for.